### PR TITLE
[Feature] Support mim download odl datasets

### PR DESCRIPTION
**Pull request by Tau-J**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: tools/dataset_converters/scripts/preprocess.aic.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```
Comment (ly015) [inline tools/dataset_converters/scripts/preprocess.aic.sh:1]: The file name should be preprocess_aic.sh?

File: docs/zh_cn/user_guides/prepare_datasets.md [hunk 2]
```diff
@@ -1,264 +1,221 @@
...
-        'bbox': [402.34, 205.02, 65.26, 88.45],
-        'category_id': 1,
-        'id': 215218
-    },
-    ...
-],
-'categories': [
-    {'id': 1, 'name': 'person'},
- ]
+In this document, we will give a guide on the process of preparing datasets for the MMPose. Various aspects of dataset preparation will be discussed, including using built-in datasets, creating custom datasets, combining datasets for training, browsing and downloading the datasets.
```
Comment (Ben-Louis) [inline docs/zh_cn/user_guides/prepare_datasets.md:3]: There seem to be some problems with the modification of this file

File: docs/en/user_guides/prepare_datasets.md [hunk 3]
```diff
@@ -155,3 +155,67 @@ Here is a processed sample
...
+
+# install OpenDataLab CLI tools
+pip install -U opendatalab
+# log in OpenDataLab, registry
+odl login
+
+# download coco2017 and preprocess by MIM
+mim download mmpose --dataset coco2017
+```
+
```
Comment (Ben-Louis) [inline docs/en/user_guides/prepare_datasets.md:177]: Perhaps we could include an example command for extracting data from the downloaded file using the new scripts?

Comment (Tau-J) [reply docs/en/user_guides/prepare_datasets.md:177]: The preprocessing script will be automatically executed by mim

**Review by Ben-Louis — APPROVED** [review]

**Pull request merged**